### PR TITLE
Soft 84 build footer of the website

### PR DESCRIPTION
**Pull request by JustYousef2024**
Finish Website Footer

### CODE REVIEW

File: customHooks/Auth/useLogin.tsx
Comment (MuhammadSawalhy) [inline customHooks/Auth/useLogin.tsx:1]: Is moving hooks in `customHooks` directory to their corresponding page in the `app` directory

File: components/Settings/settingStyle.css
Comment (MuhammadSawalhy) [inline components/Settings/settingStyle.css:1]: Why not Tailwind?

File: tsconfig.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "**/**/*.ts",
     "**/**/*.tsx",
     ".next/types/**/*.ts",
-    "components/Navbar/guestNavbarStyle.jsx",
+    "components/Navbar/guestNavbarStyle.jsx", "components/Navbar/ProfileDropdown.tsx",
```
[the remark this answers is not in the document]
Comment (MuhammadSawalhy) [reply tsconfig.json:32]: Are `**/*.ts` and `**/*.tsx` enough?

[the remark this answers is not in the document]
Comment (JustYousef2024) [reply tsconfig.json:32]: I think it is enough. I didn't add this line of code, I don't know why it written.

File: api/nonAuthRequest.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 // env didn't work properly with codespace, if run local uncoment next line and comment the next 2 line.
 // const baseUrl = process.env.BASEURL
-const baseUrl = 'https://mohamedham.pythonanywhere.com';
+const baseUrl = 'https://ikseer.onrender.com';
```
[the remark this answers is not in the document]
Comment (JustYousef2024) [reply api/nonAuthRequest.tsx:5]: env work with local workspace but didn't work as expected with codespaces

**Comment by JustYousef2024** [comment]
- Previous issues have been solved, Review if any outstanding issues need resolution

**Comment by JustYousef2024** [comment]
issues Solved & resolve conflicts.

**Pull request merged**